platform/osx/os_osx.h

```diff
@@ -31,6 +31,8 @@
 #ifndef OS_OSX_H
 #define OS_OSX_H
 
+#define BitMap _QDBitMap // Suppress deprecated QuickDraw definition.
```

Comment:
FWIW according to my understanding identifiers that begin with an underscore & a capital letter are reserved--their use is considered undefined behavior.

I think it would be more "correct" to just use `QDBitMap` here.

References:

 * "the identifiers that begin with an underscore followed by an uppercase letter are reserved; " in point 3 [here](https://en.cppreference.com/w/cpp/language/identifiers#In_declarations).
 * Point 3 [here](https://en.cppreference.com/w/c/language/identifier#Reserved_identifiers).
 * [GNU C library documentation](https://www.gnu.org/software/libc/manual/html_node/Reserved-Names.html).
 * "[use _ and __ in C programs](https://stackoverflow.com/questions/25090635/use-and-in-c-programs#25090719)" post.